modules/gateway/peers.go

```diff
@@ -26,10 +24,37 @@ const (
 	minNodeListLen = 100
 )
 
+var (
+	// The gateway will sleep this long between incoming connections.
+	acceptInterval = func() time.Duration {
```

Comment:
the delay only applies when you are trying to connect to a peer, not when sending an RPC. So it shouldn't slow down block/txn propagation.